### PR TITLE
upgrade Selenium version to 3.12.0

### PR DESCRIPTION
**Pull request by Chr1st0ph**
upgrade Selenium version to 3.12.0

@flah00 @diemol fyi

### CODE REVIEW

File: stable/selenium/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: selenium
-version: 0.3.1
-appVersion: 3.11.0
+version: 0.3.2
```
Comment (rjkernick) [inline stable/selenium/Chart.yaml]: Please bump the minor version instead

**Review by rjkernick — CHANGES_REQUESTED** [review]

File: stable/selenium/Chart.yaml (hunk 1 above)
Comment (Chr1st0ph) [reply stable/selenium/Chart.yaml]: @rjkernick I updated the minor version as requested. I think there is a lack of definition concerning the usage of Semantic Versioning ( previous updates of Selenium version have been "marked" via patch version in Chart.yaml )

**Review by rjkernick — APPROVED** [review]

**Pull request merged**